lib/idpartner.js

```diff
@@ -180,7 +180,7 @@ class IDPartner {
       identity_provider_id: query.idp_id,
       idpartner_token: idpartnerToken,
       claims: JSON.stringify(claimsObject),
-      ...(this.config.token_endpoint_auth_method === 'client_secret_basic' ? { client_secret: this.config.client_secret } : { response_mode: 'jwt' }),
```

Comment:
why are we deleting the client secret from here? Would client secret work after these changes?

Reply:
yes it works